pages/docs/tutorials/kafka/configure-kafka-avro.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Create AsyncAPI Document for applications interacting with Kafka using Avro Schema
+description: Explore configuring AsyncAPI document for Kafka messages with Avro Schema.
+weight: 30
+---
+
+## Introduction
+
+The previous tutorial taught you about writing an AsyncAPI document for Kafka messages using the default schema. This turorial will teach you to write the same document using Avro Schema.
+
+While AsyncAPI schemas can be the default choice for describing payloads, many prefer using Avro Schemas to define messages in Kafka. Through this tutorial you'll learn to modify your existing AsyncAPI schema to add Avro schema into your document in both YAML and JSON formats.
+
+## Background Context
+
+AsyncAPI is a specification for describing Event-Driven Architectures (EDAs) in a machine-readable format. It allows developers to define how messages are exchanged between different services by providing a agreement for communication.
+
+Avro is an efficient binary serialization format, used to ensure schema-aware communications between the messages within Apache Kafka. With Avro Schema there is a standardized method for serializing data allowing for interoperability and schema evolution capabilities. Avro promotes data exchange among systems by offering a common schema that fosters compatibility, between different components. 
+
+## Defining messages using Avro Schema in YAML
+
+In this section, let's focus on defining messages using Avro Schemas using YAML. 
```

Comment:
since this tutorial is continuation of the previous one you wrote, you should like to it + kinda explain that in previous tutorial you explained how to use kafka with asyncapi but that it was done using default AsyncAPI features, so for schema AsyncAPI Schema (superset of JSON Schema) was used and that here you rework schemas. I think would be good to show asyncapi file from previous tutorial, at least part with schemas

Reply:
I have added a reference to the previous tutorial in the Introduction. Also, your suggestion about adding the message schema from the previous tutorial sounds good, I'll add it right away.